Rocket.Chat/Resources/pl.lproj/Localizable.strings

```diff
@@ -184,6 +184,8 @@
 
 // Members List
 "chat.members.list.title" = "Członkowie";
+"chat.member.unblock.title" = "Odblokować";
```

Comment:
„Odblokuj”

Reply:
Ok, thanks. But I cannot pass the test))

Reply:
Do not worry about diff coverage too much

Reply:
And how can I know, that my changes were accepted?

Reply:
They will be reviewed and merged by owners unless they require some changes